app/controllers/items_controller.rb

```diff
@@ -18,13 +22,14 @@ def destroy
 
     if @item.destroy
       flash[:notice] = "Item was deleted."
+    #  redirect_to user_path(current_user)
     else
       flash[:alert] = "Item couldn't be deleted. Please try again."
-
-      respond_to do |format|
-        format.html
-        format.js
-      end
+    #  redirect_to user_path(current_user)
     end
+    #respond_to do |format|
```

Comment:
Commenting this out is partially causing the missing template error. 
If you are not going to redirect like in `#create`.. you need a `destroy` view to render.  This code instructs it to render that view in `html` or `js` depending on what the browser requested. 

This will likely fail if the browser requests html, because there is no `items/destroy.html.erb` view.  Note, I am not saying that is the desired behavior.. just that is why its failing.  If the browser requests `JS`, it should respond with your `items/destroy.js.erb` view.